src/memory/server/MemoryServer.ts

```diff
@@ -13,19 +13,17 @@ import { ClientRequest, ServerResponse, ReadMemory } from '../common/messages';
 import { MemoryContents } from 'cdt-gdb-adapter';
 
 export class MemoryServer {
-    private panel?: vscode.WebviewPanel;
 
     constructor(context: vscode.ExtensionContext) {
         context.subscriptions.push(
             vscode.commands.registerCommand('cdt.gdb.memory.open', () =>
-                this.openPanel(context)
+                this.openPanel(context);
```

Comment:
This change introduced a syntax error - I have just pushed a fix to your branch rather than hold up the PR.

I hope to get GitHub actions installed on this repo soon too so that next time you will get a fairly quick analysis of this rather than having to wait for full review.